File: jobs/2026-06-08__12-44-45/encore_create_wasp_hello_world_ts__u2bJsi3/artifacts/user/myproject/hello/hello.ts
import { api } from "encore.dev/api";

interface Response {
  message: string;
}

// Returns a personalized greeting.
export const get = api(
  { expose: true, method: "GET", path: "/hello/:name" },
  async ({ name }: { name: string }): Promise<Response> => {
    const msg = `Hello ${name}!`;
    return { message: msg };
  }
);

File: jobs/2026-07-02__17-09-42/encore_create_hello_world_go__8fwDpmv/artifacts/user/helloworld-zrs46h5gyp/hello/hello.go
package hello

import (
	"context"
	"fmt"
)

// WorldResponse is the JSON response returned by the World endpoint.
type WorldResponse struct {
	Message string `json:"message"`
}

// World returns a friendly greeting for the provided name.
//
//encore:api public path=/hello/:name method=GET
func World(ctx context.Context, name string) (*WorldResponse, error) {
	msg := fmt.Sprintf("Hello, %s!", name)
	return &WorldResponse{Message: msg}, nil
}